linera-core/src/remote_node.rs

```diff
@@ -88,14 +106,56 @@ impl<N: ValidatorNode> RemoteNode<N> {
         self.check_and_return_info(response, chain_id)
     }
 
-    pub(crate) async fn handle_optimized_certificate<T: ProcessableCertificate>(
+    pub(crate) async fn handle_optimized_timeout_certificate(
```

Comment:
Let's get rid of that and send the full certificate right away for timeouts. They are small. (Let's also make sure we don't cache them.)

Reply:
Separate PR, ok? I don't want to mix in logic changes.

Reply:
Done in [Send full timeout certificate (not lite).](https://github.com/linera-io/linera-protocol/pull/2996/commits/934fd93121af1d5ba9b81433a8bbadadb49bf9dd). 

Turned out to be easy and small.